### DISCUSSION TITLE
What is a PB&J?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#953] [Galaxy Trivia] [answered]
**Bonus:** Who invented it?

**Comment by peterbaccaro** [comment, marked as answer]
A PB&J is an acronym for a **peanut butter and jelly sandwich**. It is a classic and popular sandwich in many parts of the world, particularly in North America. It typically consists of two slices of bread with peanut butter spread on one side and jelly or jam spread on the other. The peanut butter and jelly combination creates a sweet and savory taste that is enjoyed by many people, especially children. PB&J sandwiches are often eaten as a quick and easy snack or packed in lunches for school or work.

In 1901, the first peanut butter and jelly sandwich recipe appeared in the Boston Cooking School Magazine of Culinary Science and Domestic Economics written by **Julia Davis Chandler**. She said to use currant or crab-apple jelly and called the combination delicious and as far as she knew, original.
